### ISSUE TITLE
[Feature]: Add plain React + Vite example in CodeSandbox

### ISSUE THREAD

**Issue by Mikilll94** [fabricjs/fabric.js#9705]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have searched and referenced existing issues, feature requests and discussions
- [X] I am filing a **FEATURE** request.

### Description

Add plain React + Vite example in CodeSandbox. Not everyone uses Next.js.

### Current State

Currently there is only Next.js example in CodeSandbox.

**Comment by ShaMan123** [comment]
I am not a vite dev
Are you willing to PR?
I did a vue example not long ago
Also look at #9700 for the react hooks

**Comment by ShaMan123** [comment]
Isn't vue enough?

**Comment by asturur** [comment]
Isn't vitejs just a simple plain framework to start a react application, a sort of modern create-react-app ?

**Comment by asturur** [comment]
i used it recently is just plain react with nothing more ( configured eslint, typescript and css yes )

**Comment by Mikilll94** [comment]
> Isn't vitejs just a simple plain framework to start a react application, a sort of modern create-react-app ?
> 
> i used it recently is just plain react with nothing more ( configured eslint, typescript and css yes )

yeah, that's true. Create-react-app is no longer maintained and recommended for developing React. Vite is the most popular and sensible for choice for starting react project now.

**Comment by rk9155** [comment]
use fabric version "6.0.0-beta19" it is in beta but works well with vite no extra setup needed just added it and start trying basic components from docs.
Reason: Current fabric stable version 5.3.1 won't work with esm6 module support.

**Comment by Shivamc489** [comment]
I have been using this package for my react + vite app
see if this might help you as well
https://www.npmjs.com/package/fabricjs-react-improve

**Comment by vaynevayne** [comment]
Recommend https://github.com/cs-open/react-fabric

**Comment by asturur** [comment]
@vaynevayne opened 3 emails and in all of them was you linking react-fabric.
Please don't.
Is linked here already: https://fabricjs.com/resources/
